meson.build

```diff
@@ -23,8 +23,16 @@ config_h = configure_file(
   output: 'config.h',
 )
 
+drm_fourcc_header_path = join_paths(libdrm.get_pkgconfig_variable('includedir'), 'libdrm/drm_fourcc.h')
+pymod = import('python')
```

Comment:
>The build bot doesn't seem to be able to import python module but it builds fine here.

It's so frustrating how old some debian packages can be. Can you change this to the (now deprecated) `python3` module instead?

We'll change back it when the new debian stable is released.

Reply:
I did but it didn't seem to help.

Reply:
God damn it debian...
They have 0.37 in their repos, and the python3 module was added in 0.38.
I suppose it'll just have to be changed back to how this originally was, without the meson modules.

Reply:
Done